src/WasmParser.ts

```diff
@@ -2371,7 +2371,7 @@ export class BinaryReader {
       this._pos = pos;
       return false;
     }
-    var payloadLength = this.readVarUint32();
+    var payloadLength = this.readVarUint32() >>> 0;
```

Comment:
Shouldn't `readVarUint32()` take care of the signedness?

Reply:
+1 there are quite some places where we use `readVarUint32` without checking the signedness, so +1 to moving it there.